README.md

```diff
@@ -10,12 +10,18 @@ This repository hosts the Kubernetes Python Operator for OIDC Gatekeeper
 
 The OIDC Gatekeeper Operator may be deployed using the Juju command line as follows
 ```bash
-juju deploy oidc-gatekeeper
-juju config oidc-gatekeeper client-secret=<client-secret> public-url=http://<public-url>
```

Comment:
This config option is still there, so are we going to remove it or will we go with Deprecation warning as we did in dex-aut?

Reply:
Please check #163, I think there is enough context there